app/models/Preset.java

```diff
@@ -37,17 +37,19 @@
     @ManyToOne(cascade = CascadeType.REFRESH)
     public Camera camera;
 
-    @Constraints.Required
-    public int pan;
-
-    @Constraints.Required
-    public int tilt;
+    @JsonIgnore
+    @ManyToOne(cascade = CascadeType.REFRESH)
+    public Script script;
 
-    @Constraints.Required
-    public int zoom;
+    /**
+     * Used to link to a real world preset.
+     */
+    public int realPresetId;
```

Comment:
I am for the dummy/real term. linkedCameraPreset does not imply any external preset but a local one. 😜